options/locale/locale_en-US.ini

```
@@ -614,6 +621,9 @@ forks = Forks
 pick_reaction = Pick your reaction
 reactions_more = and %d more
 
+template.items = Template Items
+template.git_content = Git Content
```

```suggestion
template.git_content = Git Content from default branch
```
I'd rather have all branches copied, but if it's only one, I'd say it here.

---

The thing is, branches will rely on commits that won't exist after generation.  
I'm not sure how that will be accomplished.  
  
But I think the suggestion is a good one.